src/Strings.js

```diff
@@ -0,0 +1,113 @@
+module.exports = {
+  // Genates a CLI string for the current sequence
+  toCliString : function() {
+    var cliStringSteps = `"`, cliOptions = {};
+    for (var step in this.steps) {
```

Comment:
Instances of `this` would become `sequencer`

Reply:
Or perhaps we'd pass in just `steps` and the other necessary references, instead of the whole sequencer.